packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -102,6 +104,12 @@ class InteractiveViewer extends StatefulWidget {
            && boundaryMargin.left.isFinite)),
        super(key: key);
 
+    /// The content will be clipped (or not) according to this option.
+  /// See the enum [Clip] for details of all possible options and their common use cases.
+  ///
```

Comment:
I think we should reiterate the warning from the docs above, so I say add another paragraph here:

"
If set to [Clip.none], the child may extend beyond the size of the InteractiveViewer, but it will not receive gestures in these areas.  Be sure that the InteractiveViewer is the desired size when using [Clip.none].
"